packages/kit/src/runtime/client/client.js

```diff
@@ -1311,6 +1310,21 @@ export function create_client({ target }) {
 		after_navigate();
 	}
 
+	/**
+	 * @param {unknown} error
+	 * @param {import('types').NavigationEvent} event
+	 * @returns {import('../../../types/private.js').MaybePromise<App.Error>}
```

Comment:
```suggestion
	 * @returns {import('types').MaybePromise<App.Error>}
```

Reply:
Unrelated to this path change but since handle_error doesn't await the user's handleError, if they have an async one that doesn't return anything it might break stuff because `Promise<undefined>` passes the `??`,

I checked the client file and there isn't anywhere that's affected by that currently but is that something to be defensive about or would it be considered user error?